vendor/github.com/keybase/gomounts/windows.go

```diff
@@ -0,0 +1,40 @@
+// +build windows
+
+package gomounts
+
+/*
+#include <string.h>
+#include <stdlib.h>
+#include <Windows.h>
+*/
+import "C"
+
+import (
+	"unsafe"
+)
+
+// Windows implementation
+func getMountedVolumes() ([]Volume, error) {
+	result := make([]Volume, 0)
+	var buf [256]C.char
+
+	drives := uint32(C.GetLogicalDrives())
```

Comment:
hmm, isn't it possible for mounts to be places other than `<LETTER>:\`? @zanderz do you know?

Reply:
Not that it matters too much, since we won't be running this on windows